php/commands/site.php

```diff
@@ -612,76 +612,82 @@ public function init_env($args, $assoc_args) {
    }
 
   /**
-   * Clone dev to test or test to live
-   *
-   * ## OPTIONS
-   *
-   * [--site=<site>]
-   * : Site to use
-   *
-   * [--from-env]
-   * : Environment you want to clone from
-   *
-   * [--to-env]
-   * : Environment you want to clone to
-   *
-   * [--db]
-   * : Clone the database? (bool) default no
-   *
-   * [--files]
-   * : Clone the files? (bool) default no
-   *
-   * @subcommand clone-env
-   */
-   public function clone_env($args, $assoc_args) {
-     $site = SiteFactory::instance( Input::sitename( $assoc_args ) );
-     $site_id = $site->getId();
-     $from_env = Input::env($assoc_args, 'from-env', "Choose environment you want to clone from");
-     $to_env = Input::env($assoc_args, 'to-env', "Choose environment you want to clone to");
-
-     $db = $files = false;
-     $db = isset($assoc_args['db']) ?: false;
-     $append = array();
-     if ($db) {
-       $append[] = "DATABASE";
-     }
-     $files = isset($assoc_args['files']) ?: false;
-     if ($files) {
-       $append[] = 'FILES';
-     }
-     $append = join(' and ', $append);
+    * Clone dev to test or test to live
+    *
+    * ## OPTIONS
+    *
+    * [--site=<site>]
+    * : Site to use
+    *
+    * [--from-env]
+    * : Environment you want to clone from
+    *
+    * [--to-env]
+    * : Environment you want to clone to
+    *
+    * [--db-only]
+    * : Clone only the the database
+    *
+    * [--files-only]
+    * : Clone only the files
+    *
+    * @subcommand clone-content
+    */
+  public function clone_content($args, $assoc_args) {
+    $site     = SiteFactory::instance(Input::sitename($assoc_args));
+    $site_id  = $site->getId();
+    $from_env = $site->environment(Input::env(
+      $assoc_args,
+      'from-env',
+      'Choose environment you want to clone from'
+    ));
+    $to_env   = Input::env(
+      $assoc_args,
+      'to-env',
+      'Choose environment you want to clone to'
+    );
 
-     if (!$files AND !$db) {
-       \Terminus::error('You must specify something to clone using the the --db and --files flags');
-     }
+    $db     = isset($assoc_args['db-only']);
+    $files  = isset($assoc_args['files-only']);
+    $append = array();
+    if ($db) {
+      $append[] = "DATABASE";
+    }
+    if ($files) {
+      $append[] = 'FILES';
+    }
+    $append = join(' and ', $append);
 
-     $confirm = sprintf("Are you sure?\n\tClone from %s to %s\n\tInclude: %s\n", strtoupper($from_env), strtoupper($to_env), $append);
-     \Terminus::confirm($confirm);
+    if (!$files && !$db) {
+      $files = $db = true;
+    }
```

Comment:
Does this block need to go above the append string so it's accurate for the default case of both?